data/net/minecraft/world/level/levelgen/Beardifier.mapping

```diff
@@ -1,8 +1,26 @@
 CLASS net/minecraft/world/level/levelgen/Beardifier
+	COMMENT Modifies terrain noise to be flatter near structures such as villages.
+	COMMENT It's called a beardifier since it looks like Jeb's beard, the bottom part is solid and the top part is shaved.
```

Comment:
Okey out of pure curiosity.... Source?

Reply:
Uhh... source said it was a joke and I'm just dumb :P